docs/version0.33.md

```diff
@@ -47,6 +48,9 @@ This was already required for OpenJDK 17 and is now also required from 11.0.16 t
 
 Linux builds for all platforms now use gcc 10.3 instead of gcc 7.5. See the list of [build environments](openj9_support.md#build-environments).
 
+### Control groups v2 support
+
+The Linux kernel has two variants of [control groups (cgroups): v1 and v2](https://man7.org/linux/man-pages/man7/cgroups.7.html). Many Linux operating systems are gradually transitioning from cgroups v1 to v2 as their default choice. Now, OpenJ9 has identical support for both cgroups v1 and v2.
```

Comment:
I suggest updating the wording to indicate v2 support was added, such as: Now, OpenJ9 has added cgroups v2 support which is identical to the support for v1.

Reply:
Updated.